src/app/admin/usersTable/UsersTable.tsx

```diff
@@ -23,13 +23,13 @@ export const OptionButtonDiv = styled.div`
     justify-content: center;
 `;
 
-const usersTableHeaderKeysAndLabels: TableHeaders = [
+const usersTableHeaderKeysAndLabels = [
     ["id", "User ID"],
     ["email", "Email"],
     ["userRole", "Role"],
     ["createdAt", "Created At"],
     ["updatedAt", "Updated At"],
-];
+] as const;
```

Comment:
https://blog.logrocket.com/complete-guide-const-assertions-typescript/, it's required so that the type isn't inferred to be `[string, string][]` but instead to a subtype of `[keyof Data, string][]`